welcome.css

```diff
@@ -57,7 +57,7 @@ html, body {
   display: flex;
   justify-content: center;
   align-items: center;
- }
+}
 
 .menu-list a:hover { 
```

Comment:
Please write all styles related to `.menu-list `together. It's hard to find.

##### Not Good (menu-list is Separated)
```
.menu-list li:last-child {
}
.menu-item {
}
. menu-list a:hover {
}
```
##### Good
```
.menu-list {
}
.menu-list li {
}
.menu-list a:hover {
}
```